src/ui-manager.tsx

```diff
@@ -164,10 +164,11 @@ class UIManager {
    * @returns {void}
    */
   _createStore(config: UIOptionsObject): void {
-    // const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
-    // TODO - fix the window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
-    // TODO - Deprecated symbol used, consult docs for better alternative, React recommend using the configureStore method of the @reduxjs/toolkit packag
-    this.store = createStore<RootState, any, any, any>(reducer, compose(middleware(this.player, config)));
+    const reduxDevtoolExtension = (window as any).__REDUX_DEVTOOLS_EXTENSION__;
```

Comment:
i know why this is needed, but it should be in a separate PR

if someone looks at this line in the future they should know why this code was changed

Reply:
changes moved to https://github.com/kaltura/playkit-js-ui/pull/863